pandas/tseries/frequencies.py

```diff
@@ -95,16 +95,18 @@ def to_offset(freq) -> Optional[DateOffset]:
 
     Examples
     --------
-    >>> to_offset('5min')
+    >>> import datetime
```

Comment:
timedelta is in the module namespace.  instead of importing here, would this be fixed by changing `datetime.timedelta` on L110 to just `timedelta`?

Reply:
```pytest``` is having a scope for each docstring, so in order to have ```timedelta``` instead of ```datetime.timedelta```, L98 would have to be ```from datetime import timedelta```.

Another option (which IMO is least preferred, is to import it below, at L109 for example)

Reply:
we always have `pd` in the namespace right?  how about `pd.Timedelta`

Reply:
> we always have `pd` in the namespace right? how about `pd.Timedelta`

Worked!